packages/core/schematics/migrations/typed-forms/util.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import ts from 'typescript';
+
+import {getImportSpecifier} from '../../utils/typescript/imports';
+import {isReferenceToImport} from '../../utils/typescript/symbol';
+
+export const controlClassNames =
+    new Set(['AbstractControl', 'FormArray', 'FormControl', 'FormGroup']);
+export const builderMethodNames = new Set(['control', 'group', 'array']);
+export const anySymbolName = 'AnyForUntypedForms';
+
+export interface MigratableNode {
+  node: ts.Expression;
+  generic: string;
+}
+
+export function getControlClassImports(sourceFile: ts.SourceFile) {
+  return Array.from(controlClassNames)
+      .map(cclass => getImportSpecifier(sourceFile, '@angular/forms', cclass))
```

Comment:
- since the `controlClassNames` is used once, we can just make it an array and drop `Array.from`
- I'd propose making the name of the function a bit more descriptive 
```suggestion
export function getAbstractControlBasedClassImports(sourceFile: ts.SourceFile) {
  return controlClassNames
      .map(cclass => getImportSpecifier(sourceFile, '@angular/forms', cclass))
```

Reply:
Done.